src/components/themes/purple-haze/theme.css

```diff
@@ -569,3 +569,7 @@ a[data-role=button] {
     background-color: #dbe6ff;
     color: #0e0f2d;
 }
+
+.metadataSidebarIcon {
+    color: #dbe6ff
```

Comment:
Will a color need added to the other themes for the icons?

Reply:
@thornbill it's done add to other `#00a4dc`